ui/pages/base.rb

```diff
@@ -6,7 +6,34 @@ def self.set_url(path)
       super(ENV['BASE_URL'] + path)
     end
 
+    def initialize(default_params = {})
+      @default_params = default_params
+      super()
+    end
+
+    def wait(*args, &block)
+      SitePrism::Waiter.wait_until_true(*args, &block)
+    end
+
+    def displayed?(*args)
```

Comment:
What does this do?

Reply:
It checks that the page is displayed in the script.

Reply:
There is an example of it being used on the question page

```
def answer_with(response, args = {})
      wait_until_radio_buttons_visible
      choose(response)
      next_button.click
      sleep(2)
      QuestionPage.new(args).tap do |page|
        wait { page.displayed? }
      end
    end
```

Reply:
The flakiness might be because the review app is set to deploy YAML following Helen's last merge, we didn't revert her review app config change.